tests/json_to_stix/test_class.py

```diff
@@ -48,19 +48,31 @@ def test_observation_prop(self):
                ['name'] == datasource['name'])
 
     def test_simple_props(self):
-        datasource = {'id': '123', 'name': 'sourcename'}
-        map_data = {"ip": {
-            "key": "ipv4-addr.value",
-            "type": "value"
-        }, "url": {"key": "url.value",
-                   "type": "value"},
-            "domain": {
-            "key": "domain-name.value",
-            "type": "value"
-        }, "payload": {
-            "key": "artifact.payload_bin",
-            "type": "value"
+        datasource = {
+            'id': '123',
+            'name': 'sourcename'
         }
+        map_data = {
+            "ip": {
```

Comment:
I know this was previously in the test, and won't even matter if we just use the existing map file for testing, but ip doesn't actually come back from QRadar (or at least it's not in our mapping). This is pretty much handled by the source and destination ip tests anyway.